as2_motion_controller/include/as2_motion_controller/controller_handler.hpp

```diff
@@ -167,7 +168,7 @@ class ControllerHandler
   geometry_msgs::msg::TwistStamped state_twist_;
   geometry_msgs::msg::PoseStamped ref_pose_;
   geometry_msgs::msg::TwistStamped ref_twist_;
-  as2_msgs::msg::TrajectoryPoint ref_traj_;
+  as2_msgs::msg::Thrust ref_thrust_;
```

Comment:
Trajectory motion reference subscriber is still kept, so why to overwrite its msg attribute and callback?

Reply:
indeed ci fails due to these two lines, going to restore them